.ci/no_old_refs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Attention, there is no "-x" to avoid problems on Travis
+set -e
+
+MENTIONED_IDS=mentioned_ids
+CLOSED_ISSUES=failed
```

Comment:
please create such files in `.ci-temp` folder.
we try to not pollute (if script is failed in the middle, and rm is no done) in git repo even unintentionally.

Reply:
done

Reply:
May be use /tmp/ folder?

Reply:
for some reason I think  it is better to keep resources in filesystem we have - git repo.
but I do not remember exact reason, some CI might not have `/tmp`, but I do not remember which ..